src/main/java/org/javarosa/form/api/FormController.java

```diff
@@ -121,6 +122,17 @@ public void postProcessInstance() {
         mFormEntryController.getModel().getForm().postProcessInstance();
     }
 
+    @Nullable
+    public String getFormDescriptor() {
+        FormDef formDef = mFormEntryController.getModel().getForm();
+        String caseNameDescriptor = formDef.getLocalizer().getText("case-name-descriptor");
```

Comment:
We should probably move this string out to a constant. But I'm not sure where to put it. Is there a place where we keep these special itexts?